config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rs::Application.routes.draw do
 
+  require 'sidekiq/web'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.